ghc-toolkit/boot-libs/integer-simple/GHC/Integer/Type.hs

```diff
@@ -32,7 +32,7 @@ import GHC.Classes
 import GHC.Magic
 import GHC.Prim
 import GHC.Types
-
+ 
```

Comment:
It would be nice to set up some plugin to trim the trailing whitespaces in your editor, to prevent this in the future.